buildSrc/src/main/java/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -381,47 +390,68 @@ public String toString() {
         List<ElasticsearchDistribution> currentDistros = new ArrayList<>();
         List<ElasticsearchDistribution> upgradeDistros = new ArrayList<>();
 
-        for (Type type : List.of(Type.DEB, Type.RPM, Type.DOCKER)) {
-            for (Flavor flavor : Flavor.values()) {
-                for (boolean bundledJdk : Arrays.asList(true, false)) {
-                    // All our Docker images include a bundled JDK so it doesn't make sense to test without one
-                    boolean skip = type == Type.DOCKER && bundledJdk == false;
-
-                    if (skip == false) {
-                        addDistro(distributions, type, null, flavor, bundledJdk, VersionProperties.getElasticsearch(), currentDistros);
+        for (Architecture architecture : Architecture.values()) {
+            for (Type type : List.of(Type.DEB, Type.RPM, Type.DOCKER)) {
+                for (Flavor flavor : Flavor.values()) {
+                    for (boolean bundledJdk : Arrays.asList(true, false)) {
+                        // All our Docker images include a bundled JDK so it doesn't make sense to test without one.
+                        // Also we'll never publish an ARM (aarch64) build without a bundled JDK.
```

Comment:
I'd like clarification on this. Why wouldn't we? One main difference between the archives is the version of the bundled JDK but there are other differences (config changes, eventually ml binaries, etc). @jasontedor what's the rationale for offering a no-jdk distro for x86 but not arm?

Reply:
Yes, this is deliberate that we do not have a no-jdk distribution for aarch64. The reason is because we'd like to go the other direction and eventually remove the no-jdk distribution. It's easier for us to start from a position of there is no-jdk distribution and adding one if sufficient demand proves that it's needed, than it is to start from a position of having it and later trying to remove it (we are already in a difficult position with the no-jdk images for x64).